README.md

```diff
@@ -26,6 +26,10 @@ Make it possible to configure the default upload folder for a certain TCA column
     }
 ```
 
+**FAQ**
+
+- When the user does not have access to the folder the "select & upload" buttons is not available.
```

Comment:
This is not really a question. Perhaps rephrase it in the form of a question (What happens when the user does not have access to the upload folder?) and answer (The "Select & upload" button is not available) or change the header.